Microsoft.DotNet.Interactive.Rendering.Tests/FormatterTests.cs

```diff
@@ -636,7 +636,7 @@ public void FormatAllTypes_allows_formatters_to_be_registered_on_fly_for_all_typ
         }
 
         [Fact]
-        public void FormatAllTypes_does_not_reregister_formatters_for_types_having_special_default_formatters()
```

Comment:
Please revert the name change. The idea is that it doesn't overwrite a pre-existing registration.